sdk/cognitiveservices/Knowledge.QnAMaker/tests/BaseTests.cs

```diff
@@ -6,53 +6,52 @@ namespace QnAMaker.Tests
     public abstract class BaseTests
     {
         public static bool IsTestTenant = false;
+        private static readonly string QnAMakerPreviewSubscriptionKey;
```

Comment:
why to add this new preview subscription key?, existing subscriptionkey not sufficient?